src/services/webclient.ts

```diff
@@ -3108,6 +3108,11 @@ export class WebClientService {
 
         // Get receiver
         const receiver = this.receivers.getData({type: data.type, id: data.id});
+        if (!hasValue(receiver)) {
+            const log = `Received conversation update for unknown receiver (${data.type}/${data.id})`;
```

Comment:
Would leak the receiver's id in the reported log.

Reply:
Fixed, good catch.